src/PreferencesDialog.cpp

```diff
@@ -42,6 +42,13 @@ PreferencesDialog::PreferencesDialog(QWidget* parent, Tabs tab)
     ui->checkUpdates->setVisible(false);
 #endif
 
+#ifdef Q_OS_MACX
+    listUpBuiltinExtension();
+#else
+    ui->labeBuiltinExtensions->setVisible(false);
```

Comment:
Minor typo?  Guessing this should be `label` rather than `labe`? :wink:

Reply:
Thank you. I just fixed.